plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -205,8 +206,14 @@ public void storeMergedRows(Page page)
     {
         DeltaLakeMergePage mergePage = createPages(page, tableColumnCount);
 
-        mergePage.insertionsPage().ifPresent(insertPageSink::appendPage);
-        mergePage.updateInsertionsPage().ifPresent(insertPageSink::appendPage);
+        mergePage.insertionsPage().ifPresent(insertions -> {
+            insertPageSink.appendPage(insertions);
+            writtenBytes += insertions.getSizeInBytes();
+        });
+        mergePage.updateInsertionsPage().ifPresent(updateInsertions -> {
+            insertPageSink.appendPage(updateInsertions);
+            writtenBytes += updateInsertions.getSizeInBytes();
+        });
```

Comment:
We should be tracking only physical written bytes that is available in `insertPageSink.getCompletedBytes()`.
So this should be addition of one line at the end of this method:
`writtenBytes = insertPageSink.getCompletedBytes()`